roles/pulp3-devel/tasks/main.yml

```diff
@@ -0,0 +1,108 @@
+---
+- block:
+
+  - name: Install several useful packages
+    package:
+      name:
+        - bash-completion
+        - dnf-utils
+        - dstat
+        - fd-find
+        - fedora-easy-karma
+        - fpaste
+        - fzf
+        - git
+        - htop
+        - httpie
+        - iotop
+        - jnettop
+        - jq
+        - ncdu
+        - python-django-bash-completion
+        - python-ipdb
+        - python-rpdb
+        - python3-setuptools
+        - python3-virtualenvwrapper
+        - redhat-lsb-core
+        - ripgrep
+        - screen
+        - telnet
+        - tmux
+        - tree
+        - wget
+        - vim-enhanced
```

Comment:
+1 leave it in +0 take it out. vi is already installed, and this package can be somewhat slow to install and has caused problems in the past.

Reply:
Since the source is mounted into the Vagrant box, everyone can use their own favourite editor (with their systemwide configuration). I'd say we don't need this.

Reply:
Reducing to a minimal set of installed packages has been moved to 4322:
https://pulp.plan.io/issues/4322
And I have a PR up here where we can continue this discussion of packages:
https://github.com/pulp/ansible-pulp3/pull/63